stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -30,18 +29,43 @@
   };
 
   private final Context mContext;
+  private final DatabaseFilesProvider mDatabaseFilesProvider;
 
-  public DatabasePeerManager(Context context) {
+  /**
+   * Constructs the object with a {@link DatabaseFilesProvider} that supplies the database files
+   * from {@link Context#databaseList()}.
+   *
+   * @param context the context
+   */
+  public DatabasePeerManager(final Context context) {
+    this(context, new DatabaseFilesProvider() {
+      @Override public Set<File> getDatabaseFiles() {
+        Set<File> databaseFiles = new HashSet<File>();
```

Comment:
Prefer a list.  `HashSet` has ordering and performance implications that are seemingly unnecessary here.  Note that Android uses a list itself assuredly for this same reason.